<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmpClientGui</class>
 <widget class="QMainWindow" name="DmpClientGui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>745</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="text">
         <string>Play/Pause</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="muteButton">
        <property name="text">
         <string>Mute</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QToolButton" name="addRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="deleteRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tuneInRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>TuneIn</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="DmpClientGuiRadioList" name="radio_list">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>636</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="DmpClientGuiPlaylists" name="playlists">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="DmpClientGuiSearchResults" name="search_results">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="DmpClientGuiSearchBar" name="search_bar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Default text</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="DmpClientGuiMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuIndex">
    <property name="title">
     <string>Index</string>
    </property>
    <addaction name="actionIndex_Folder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuIndex"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionIndex_Folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Index Folder</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DmpClientGuiSearchBar</class>
   <extends>QLineEdit</extends>
   <header>dmp_client_gui_search_bar.hpp</header>
   <slots>
    <slot>searchBarReturned()</slot>
    <slot>cursorChanged(int,int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DmpClientGuiMenuBar</class>
   <extends>QMenuBar</extends>
   <header>dmp_client_gui_menu_bar.hpp</header>
   <slots>
    <slot>indexFolder()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DmpClientGuiSearchResults</class>
   <extends>QTableView</extends>
   <header>dmp_client_gui_search_results.hpp</header>
   <slots>
    <slot>currentActiveRadio(std::string)</slot>
    <slot>queueRequest(QModelIndex)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DmpClientGuiRadioList</class>
   <extends>QListView</extends>
   <header>dmp_client_gui_radio_list.hpp</header>
   <slots>
    <signal>currentlySelectedRadio(std::string)</signal>
    <signal>setTuneInChecked(bool)</signal>
    <signal>setPlayingChecked(bool)</signal>
    <slot>addRadio()</slot>
    <slot>radioSelected(QModelIndex)</slot>
    <slot>tuneIn(bool)</slot>
    <slot>removeRadio()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DmpClientGuiPlaylists</class>
   <extends>QTableView</extends>
   <header>dmp_client_gui_playlists.hpp</header>
   <slots>
    <slot>currentRadioChanged(std::string)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DmpClientGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionIndex_Folder</sender>
   <signal>triggered()</signal>
   <receiver>menu_bar</receiver>
   <slot>indexFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_bar</sender>
   <signal>returnPressed()</signal>
   <receiver>search_bar</receiver>
   <slot>searchBarReturned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>691</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>691</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_bar</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>search_bar</receiver>
   <slot>cursorChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>691</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>691</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>radio_list</receiver>
   <slot>addRadio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_results</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>search_results</receiver>
   <slot>queueRequest(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_list</sender>
   <signal>currentlySelectedRadio(std::string)</signal>
   <receiver>search_results</receiver>
   <slot>currentActiveRadio(std::string)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_list</sender>
   <signal>currentlySelectedRadio(std::string)</signal>
   <receiver>playlists</receiver>
   <slot>currentRadioChanged(std::string)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>DmpClientGui</receiver>
   <slot>StopPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DmpClientGui</receiver>
   <slot>PlayPauseToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>DmpClientGui</receiver>
   <slot>NextPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tuneInRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>radio_list</receiver>
   <slot>tuneIn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_list</sender>
   <signal>setTuneInChecked(bool)</signal>
   <receiver>tuneInRadioButton</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>radio_list</receiver>
   <slot>removeRadio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>DmpClientGui</receiver>
   <slot>dmpConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DmpClientGui</receiver>
   <slot>VolumeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>muteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DmpClientGui</receiver>
   <slot>MuteToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchBarReturned()</slot>
  <slot>indexFolder()</slot>
  <slot>dmpConnect()</slot>
  <slot>StopPressed()</slot>
  <slot>PlayPauseToggled(bool)</slot>
  <slot>NextPressed()</slot>
  <slot>MuteToggled(bool)</slot>
  <slot>VolumeChanged(int)</slot>
 </slots>
</ui>
